highway_env/envs/common/observation.py

```diff
@@ -9,6 +9,8 @@
 from highway_env.vehicle.control import MDPVehicle
 from highway_env.road.graphics import WorldSurface
 
+import cv2
```

Comment:
Why do you need cv2?

Reply:
The image from the screen is 600*600 and need to be resized instead of reshape.

Reply:
Ah I see, sorry. (To avoid the resizing cost, it may be best to have the environment rendering configurations "screen_width" and "screen_height" match the desired observation shape while training).
Can you also add cv2 to the project dependencies in setup.py?